app-oauth.js

```diff
@@ -61,10 +62,11 @@ registerListeners(app);
 
 /** Start Bolt App */
 (async () => {
+  const logger = new ConsoleLogger();
```

Comment:
This is at least better than now, but if we may want to expose `app.logger` and use it here. With that, developers can control the logging level consistently

Reply:
🤩 I think that'd be most ideal! I agree that setting these levels in separate place isn't the best experience at the moment...

I'm also realizing this logger should be set to `debug` to match the `app` anyways!